### ISSUE TITLE
Add results to sotabench

### ISSUE THREAD

**Issue by Randl** [ultralytics/yolov5#38]
## 🚀 Feature
Add the model results on COCO to sotabench

## Motivation

[Sotabench](https://sotabench.com/benchmarks/object-detection-on-coco-minival) is arguably the best independent model benchmark which allows to obtain fair comparison between models. 

## Pitch

What is required is to add `sotabench.py` to the repository. The website has both guides and code examples. Existing submissions can be used as a guideline too.

**Comment by glenn-jocher** [comment]
@Randl great idea, thank you! I've not used sotabench before unfortunately, but I browsed the site just now and it seems to be a worthwhile effort. Have you done submissions to the site yourself?

**Comment by Randl** [comment]
I've done some submissions but mostly to classification leaderboard. Detection appears a bit more complicated, but you might find https://github.com/tensorpack/tensorpack/blob/master/sotabench/sotabench.py and https://github.com/Randl/DetectoRS/blob/master/sotabench.py useful.

**Comment by glenn-jocher** [comment]
their sotabench.py looks decently complicated, though I can't tell if that's simply due to the detectorrs implementation or the sotabench requirements. 

If anyone reading this has any experience submitting repositories to sotabench **we could use  help with this**. This looks like a great idea, especially since we will be updating the models as the year goes on, so it would be nice to see realtime-ish results update after applying changes, but unfortunately I won't be able to free up any time to work on it until July perhaps.

**Comment by Randl** [comment]
I don't think it should be closed

**Comment by glenn-jocher** [comment]
@Randl hey buddy. We don't have resources to do this ourselves, we are stretched very thin, but if you'd like to submit a PR we'd be happy to review!

**Comment by glenn-jocher** [comment]
@Randl I've created a sotabench.py file now in the root directory, which is essentially a slight modification of the existing test.py file. sotabench.py defaults to testing YOLOv5s. I've registered the repo on sotabench.com, but something is not working correctly, as all builds get skipped. Any ideas why this might be happening?

<img width="1194" alt="Screen Shot 2020-09-03 at 1 26 41 AM" src="https://user-images.githubusercontent.com/26833433/92091692-f0867b80-ed85-11ea-86e5-837b98a14ef5.png">
